CustomDimensions.php

```diff
@@ -267,11 +267,12 @@ public function getClientSideTranslationKeys(&$translationKeys)
         $translationKeys[] = 'CustomDimensions_CustomDimensionId';
     }
 
-    public function addConversionInformation(&$conversion, $visitInformation, Tracker\Request $request)
+    public function addConversionInformation(&$conversion, $visitInformation, Tracker\Request $request, Tracker\Action $action)
```

Comment:
Maybe better specify only `$action = null` without `Tracker\Action` as it may be sometimes not defined depending on the used piwik version etc